index.html

```diff
@@ -18,7 +18,6 @@
 </h3>
 <p>
   Tedious is pure Javascript, as are its dependencies.
-  (Well, the source is <a href="http://jashkenas.github.com/coffee-script/">CoffeeScript</a>.)
   So it should run without change on any platform where Node is supported.
```

Comment:
I was actually planning to do so... But, I'm not actually familiar enough with the build process of Tedious.

I believe you guys are using Babel under the hood to convert to ES5?  If so, the package,json is probably accurate and Node v0.10+ is probably suitable.  But, I haven't actually run any tests manually or automated to double check the accuracy of that.